{"name": "test_fail_create_task", "status": "failed", "statusDetails": {"message": "Failed: ScopeMismatch: You tried to access the function scoped fixture tasks_client with a session scoped request object. Requesting fixture stack:\ntests\\conftest.py:43:  def get_list_id(tasks_client)\nRequested fixture:\ntests\\conftest.py:22:  def tasks_client()", "trace": "ScopeMismatch: You tried to access the function scoped fixture tasks_client with a session scoped request object. Requesting fixture stack:\ntests\\conftest.py:43:  def get_list_id(tasks_client)\nRequested fixture:\ntests\\conftest.py:22:  def tasks_client()"}, "description": "Проверка ошибки при создании задачи без тела", "start": 1744825760336, "stop": 1744825760336, "uuid": "a03598e6-7512-46bf-965f-d0e8bea2be3c", "historyId": "2cbfe391f9c455de281226346fec946b", "testCaseId": "2cbfe391f9c455de281226346fec946b", "fullName": "tests.api.test_tasks.TestTasks#test_fail_create_task", "labels": [{"name": "feature", "value": "API: Работа с задачами"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "Grw-pc"}, {"name": "thread", "value": "31176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}